website/docs/advanced/code-references/github-action.md

```diff
@@ -3,7 +3,7 @@ id: github-action
 title: GitHub Action
 ---
 
-This section describes how to use ConfigCat's <a target="_blank" href="https://github.com/configcat/scan-repository">GitHub Action</a>
+This section describes how to use ConfigCat's <a target="_blank" href="https://github.com/marketplace/actions/configcat-scan-repository">GitHub Action</a>
 to scan your source code for feature flag and setting usages and upload the found code references to ConfigCat.
```

Comment:
```suggestion
to scan your source code for feature flag and setting usages and upload the found code references to ConfigCat. You can find the complete documentation of the GitHub Action [here]
```

Reply:
The same link should go to each integration page.

Reply:
Yeah, I noticed the integration links were broken, I fixed them.

Reply:
I meant a link to the integration's documentation page. Having the GitHub Actions docs, in general, is also useful. But I was missing a link to the complete docs of the integration itself in ConfigCat docs.